people.txt

```diff
@@ -0,0 +1,7 @@
+OLUWAFEMI SULE FELLOW Y
```

Comment:
**Code Quality: Nitpick**

You want to have a larger data file to ensure you're considering all the possible edge cases.